models/config.py

```diff
@@ -16,5 +16,7 @@ def get_all(self):
 	def get_pubnub_keys(self):
 		return self.base_request.request('/config', 'GET', endpoint=self.settings.get('pine_endpoint'))['pubnub']
 
-	def get_device_type(self):
-		return self.base_request.request('/config', 'GET', endpoint=self.settings.get('pine_endpoint'))['deviceTypes']
+	def get_device_types(self):
```

Comment:
I meant that you can actually cache full response from `self.base_request.request('/config', 'GET', endpoint=self.settings.get('pine_endpoint'))` and consider it not changing for the entire app lifespan. Then individual methods would only return specific keys from there. This can be done with the decorator as well, like with pubnub initialization:

```
@load_config
def get_device_types(self):
  return self._config['deviceTypes']
```